taskcluster/ci/config.yml

```diff
@@ -56,5 +56,11 @@ workers:
                 by-level:
                     "3": mozillavpn-3-signing
                     default: mozillavpn-t-signing
-scriptworker:
```

Comment:
Did u mean to delete these two lines at the end of the file?

Reply:
Yeah, they are not used anywhere. Probably a copy/paste from somewhere.